etl/steps/data/garden/met_office_hadley_centre/2023-01-17/near_surface_temperature.py

```diff
@@ -0,0 +1,51 @@
+"""Load a meadow dataset and create a garden dataset."""
```

Comment:
I'm getting different values in both versions, here's an example

```
from owid.catalog import Dataset
from etl.paths import DATA_DIR
ds = Dataset(DATA_DIR / "garden/met_office_hadley_centre/2023-01-17/near_surface_temperature")
print(ds['near_surface_temperature'].loc['Southern hemisphere'].loc[2022])

ds = Dataset(DATA_DIR / "garden/met_office_hadley_centre/2023-01-02/near_surface_temperature")
print(ds['near_surface_temperature'].loc['Southern hemisphere'].loc[2022])
```

Reply:
Thanks for pointing this out, the data for northern and southern hemisphere has changed significantly, I don't know why... I'm checking.

Reply:
Ouch, it was my mistake. It happened when I manually copied the snapshot metadata of the northern and southern files... I also accidentally copied the URLs. Thanks a lot for pointing this out!

Reply:
The new data now is identical, except for the latest data point, which has changed a bit. But this is expected.